### ISSUE TITLE
How to get UIImageView in cell message

### ISSUE THREAD

**Issue by xthaohg** [badoo/Chatto#467]
Hi guys, I have problem. I need getting UIImageView in cell of PhotoMessage. Thanks for reading

**Comment by shungz89** [comment]
Yup I was also trying to access UIImageView so that I don't have to download every single image.
But sadly they do not have DIRECT access.
